README.md

```diff
@@ -9,38 +9,42 @@ SAML: https://wiki.oasis-open.org/security/FrontPage
 ECP: http://docs.oasis-open.org/security/saml/Post2.0/saml-ecp/v2.0/saml-ecp-v2.0.html
 
 ## Setup
-To build the project, execute `gradle build` at the project root.
-The `distribution/command-line` module will contain a full package of the deployment after the build.
-
-Tests can be run with the `samlconf` script under `<PATH_TO_PROJECT>/distribution/command-line/build/distributions/samlconf-<VERSION>/bin/`.
-
-The `samlconf` script takes multiple parameters:
+To build the project:
 
-```
-NAME
-       samlconf - Runs the SAML Conformance Tests against an IdP or an SP
+    gradle build
```

Comment:
✏️ You could wrap this in the backticks to make it look more like a command.

Reply:
❓ Also, should this be `gradlew build` instead?

Reply:
Should definitely be using `gradlew` for all these commands.